staging/src/k8s.io/apiserver/pkg/server/filters/priority-and-fairness_test.go

```diff
@@ -294,14 +294,24 @@ func TestApfExecuteMultipleRequests(t *testing.T) {
 	defer cancel()
 	StartPriorityAndFairnessWatermarkMaintenance(ctx.Done())
 
+	var wg sync.WaitGroup
+	var mu sync.Mutex
+	var err error
 	for i := 0; i < concurrentRequests; i++ {
-		var err error
+		wg.Add(1)
 		go func() {
-			err = expectHTTPGet(fmt.Sprintf("%s/api/v1/namespaces/default", server.URL), http.StatusOK)
+			errLocal := expectHTTPGet(fmt.Sprintf("%s/api/v1/namespaces/default", server.URL), http.StatusOK)
+			if errLocal != nil {
+				mu.Lock()
+				err = errLocal
```

Comment:
if multiple fail, you'll only see one. Do you want to least `t.Log` all of them?

Reply:
Done.